components/builder-web/app/shared/project-settings/project-settings.component.html

```diff
@@ -1,12 +1,38 @@
 <div class="project-settings-component">
-  <div class="connect" *ngIf="!project && !connecting">
-    <button mat-raised-button color="primary" class="button" (click)="connect()" [disabled]="!hasPrivateKey">
-      Connect a plan file
-    </button>
-    <span class="no-keys" *ngIf="!hasPrivateKey">
-      <hab-icon symbol="add-circle"></hab-icon>
-      <a [routerLink]="['/origins', origin, 'keys']">Add required origin keys</a>
-    </span>
+  <h3>Default Visibility</h3>
+  <p>Set the default visibility of build artificats (.hart files) for the package. Applies to future build artifacts only. To change the visibility of the existing artifacts, see settings for each artifact in <a [routerLink]="['/pkgs', origin, name]">versions</a>.</p>
+  <hab-visibility-selector [setting]="visibility" (changed)="settingChanged($event)"></hab-visibility-selector>
```

Comment:
Suggest updating the headings within the visibility settings option to "Public artifact" and "Private artifact" since now we've differentiated "artifact" from "package".